common/src/main/java/com/microsoft/identity/common/internal/eststelemetry/IRequestTelemetryCache.java

```diff
@@ -36,17 +36,31 @@
     void saveRequestTelemetryToCache(final RequestTelemetry requestTelemetry);
 
     /**
-     * Get the telemetry from the cache
+     * Get the telemetry object from the cache
      *
      * @return a {@link RequestTelemetry} object
      */
     RequestTelemetry getRequestTelemetryFromCache();
 
+    /**
+     * Get the schema version of this telemetry object from cache
+     *
+     * @return schema version for telemetry object
+     */
+    String getTelemetrySchemaVersionFromCache();
+
+    /**
+     * Get the header string for telemetry object from cache
+     *
+     * @return header string for telemetry
+     */
+    String getTelemetryHeaderStringFromCache();
```

Comment:
Is this being used?

Reply:
No, I added them initially because I planned to use them but didn't. We may need these when we implement future schema versions, but I can remove it for now.

Reply:
Addressed